webpack.config.js

```diff
@@ -17,8 +17,8 @@ const path = require('path');
 module.exports = {
   entry: './src/index.ts',
   output: {
-    library: 'Workflows',
-    filename: './workflows.js',
+    library: 'Executions',
```

Comment:
I believe Workflows is a better name for this bundle - let's exclude `webpack.config.js` from the second library generation.

Reply:
I switched generation order, this should now be fixed.